src/fairseq2/models/conformer/convolution.py

```diff
@@ -42,6 +87,8 @@ def __init__(
             The dimensionality of the model.
         :param depthwise_kernel_size:
             The kernel size of the depthwise convolution.
+        :param depthwise_conv_is_causal:
+            Is the depthwise convolution causal or not.
```

Comment:
Nit: For boolean parameters, we typically use the convention "If ``True``, uses a causal depthwise convolution." or even better if we have a literature reference "If ``True``, uses a causal depthwise convolution as described in :cite:t`xyz`".